go.work.sum

```diff
@@ -1,2 +1,4 @@
+github.com/go-logr/logr v1.2.2 h1:ahHml/yUpnlb96Rp8HCvtYVPY8ZYpxq3g7UYchIYwbs=
```

Comment:
What are these two lines?

Reply:
They were from klog, which I removed. I don't know why it didn't get removed from the tidy. Removed.